buildSrc/src/main/kotlin/otel.publish-conventions.gradle.kts

```diff
@@ -8,7 +8,7 @@ plugins {
 publishing {
   publications {
     register<MavenPublication>("maven") {
-      if (tasks.names.contains("shadowJar") && findProperty("noShadowPublish") != "true") {
```

Comment:
Doh! Another motivation to move this and the other flags to the Java extension to make it typesafe